plugins/ssl/multi_ssl

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+multi_ssl - Plugin to monitor CERTificate expiration on multiple services and ports
+
+=head1 CONFIGURATION
+
+  [multi_ssl_*]
+    env.services www.service.tld blah.example.net_PORT
```

Comment:
With IPv6 addresses, I would expect the common approach of [wrapping the address in square brackets](https://en.wikipedia.org/wiki/IPv6_address#Literal_IPv6_addresses_in_network_resource_identifiers). At least `ssh` is doing this.
This would complicate the the port splitting slightly, but a simple `sed` expression to be applied to each token should solve this:
```
sed 's/^\(.\+\):\([0-9]\+\)$/\1 \2/'
```
But anyway: the decision is up to you.